### PR TITLE
Enhance clarity and consistency of step descriptions

### PR DESCRIPTION
**Pull request by agileshaw**
- Better indicating a timeout step with `Wait for up to...`
- Start the description of "UpgradeStep" with a verb
- No period at the end of the step description
- Quotation mark around variables like app name, unit name, model name, etc
- Other miscellaneous grammar and wording fixes

fixes: #288

### CODE REVIEW

File: cou/apps/base.py [hunk 1]
```diff
@@ -739,11 +739,15 @@ def _get_wait_step(self) -> PostUpgradeStep:
...
             description = (
-                f"Wait {self.wait_timeout}s for model {self.model.name} to reach the idle state."
+                f"Wait for up to {self.wait_timeout}s for model {self.model.name} "
+                "to reach the idle state"
             )
             apps = None
         else:
-            description = f"Wait {self.wait_timeout}s for app {self.name} to reach the idle state."
+            description = (
+                f"Wait for up to {self.wait_timeout}s for app {self.name} "
```
Comment (rgildein) [inline cou/apps/base.py:748]: Can you use `'` for app name.

File: cou/apps/core.py [hunk 2]
```diff
@@ -113,7 +113,7 @@ def _get_empty_hypervisor_step(self, unit: COUUnit) -> UnitUpgradeStep:
         :rtype: UnitUpgradeStep
         """
         return UnitUpgradeStep(
-            description=f"Check if unit {unit.name} has no VMs running before upgrading.",
+            description=f"Verify that unit {unit.name} has no VMs running",
```
Comment (rgildein) [inline cou/apps/core.py:116]: Can you use `'` for unit name.

**Review by rgildein — CHANGES_REQUESTED** [review]
Only two small change requests. Thanks

File: cou/apps/base.py (hunk 1 above)
Comment (agileshaw) [reply cou/apps/base.py]: Good point. I updated a new patch addressing all quotation mark alignment.

**Review by rgildein — APPROVED** [review]

**Review by gabrielcocenza — APPROVED** [review]

**Pull request merged**